systemtest/apmservertest/server.go

```diff
@@ -418,8 +412,8 @@ func (s *Server) consumeStderr(procStderr io.Reader) {
 
 	// Send the remaining stderr to s.Stderr.
 	procStderr = io.MultiReader(decoder.Buffered(), procStderr)
-	_, err := io.Copy(stderrPipeWriter, procStderr)
-	stderrPipeWriter.CloseWithError(err)
+	_, err := io.Copy(os.Stderr, procStderr)
+	return err
```

Comment:
`apmservertest.Server` is intended to have the sort of behaviour you might expect from `os/exec.Cmd`. i.e. it's not expected to write to `os.Stderr` unless told to. It it possible to fix the issue without changing the behaviour?

Reply:
Updated to avoid changing the current behaviour! :+1: